### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by SvbZ3r0**
This might be a hard sell considering I rewrote a lot of the code, but it is just a refactor; the underlying logic and functionality are still the same.  I've made it easier to add new sensors without having to write a new mqtt.publish everytime, thereby making the code a lot more manageable in the long run. I did rename some variables because I wasn't initially planning on submitting a PR , but that shouldn't realy affect anything. I also have made the use of single and double quotes more consistent.
Functionality-wise, I've added a clock speed sensor, the ability to enable/disable individual sensors, and made sure the program doesn't throw an exeption and quit if the broker is unavailable.
I've tested this on my setup and it works perfectly, but obviously more testing might be needed.

### CODE REVIEW

**Comment by Sennevds** [comment]
Sorry for the long wait. Looks good! Could you update the readme to change it back to this repo?

**Comment by SvbZ3r0** [comment]
Oh yeah sure! My bad lol

**Comment by Sennevds** [comment]
Okay it looks nice and is way cleaner to implement new sensors so thanks.
2 things I noticed. Power status isn't working because the config message is on the sensor topic and it's a binary_sensor.
In the readme maybe put in the table all the sensors that exist so people don't have to look in the source code (or maybe just in the settings_example.yaml file)

File: README.md [hunk 1]
```diff
@@ -42,19 +43,17 @@ You need to have at least __python 3.6__ installed to use System Sensors.
...
 | power_integer_state(Deprecated) | false | false | Deprecated
 | update_interval | false | 60 | The update interval to send new values to the MQTT broker 
-| check_available_updates | false | false | Check the # of avaiblable updates 
-| check_wifi_strength | false | false | Check the wifi strength 
-| check_wifi_ssid | false | false | Check the wifi connected SSID 
-| external_drives | false | \ | Declare external drives you want to check disk usage of (see example settings.yaml)
+| sensors | false | \ | Enable/disable individual sensors (see example settings.yaml for how-to). Default is true for all sensors.
 
 6. python3 src/system_sensors.py src/settings.yaml
 7. (optional) create service to autostart the script at boot:
```
Comment (Sennevds) [inline README.md:59]: Type: change system_sensors.service to example_system_sensors.service

Comment (Sennevds) [reply README.md:59]: (Line below of course but Github won't let me add the comment there for some reason 😃)

**Pull request merged**